### DISCUSSION TITLE
Several experiments in Pycaret

### DISCUSSION THREAD

**Discussion by pit9921** [pycaret/pycaret#1232] [Ideas]
I would like to define several setups and then run them all and the result would be be several score grids.
With `best_model = compare_models() ` it would be the best model choosen.
The same as it works with hyperparameter tuning.

The goal is to set up several normalisation techniques, several transformation, several folds, several session_ID's  etc. -  without need to run them separately.

Is there any way, how to manage it?

**Comment by pycaret** [comment]
@pit9921 Unfortunately this is not possible out-of-the-box right now but you can write a script to run multiple setups (maybe in a loop) and save the best pipeline from each run on your disk. In PyCaret 3.0 with our new API, all the preprocessing features such as `normalization` or `transformation` will also be tunable.
